webviz_subsurface/_private_plugins/tornado_plot.py

```diff
@@ -58,6 +58,7 @@ def __init__(
         )
         self.allow_click = allow_click
         self.uid = uuid4()
+        self.plotly_theme = app.webviz_settings["theme"].plotly_theme
```

Comment:
Should we instead here set
```
self.theme = app.webviz_settings["theme"]
```
in case we later need non-related plotly theme settings, and in addition it perhaps...